relay.go

```diff
@@ -281,6 +282,22 @@ func (r *Relayer) Relay(f *Frame) (shouldRelease bool, _ error) {
 	return r.handleCallReq(cr)
 }
 
+func (r *Relayer) updateCallReqContinueChecksum(f *Frame, cs Checksum) {
```

Comment:
nit: maybe rename `updateMutatedCallReqContinueChecksum`? to indicate this only needs to happen with mutation